pkg/push/api/api.go

```diff
@@ -236,20 +241,149 @@ func (s *PushService) CreatePush(
 		}
 		return nil, dt.Err()
 	}
-	return &pushproto.CreatePushResponse{}, nil
+	return &pushproto.CreatePushResponse{
+		Push: push.Push,
+	}, nil
 }
 
-func (s *PushService) validateCreatePushRequest(req *pushproto.CreatePushRequest, localizer locale.Localizer) error {
-	if req.Command == nil {
-		dt, err := statusNoCommand.WithDetails(&errdetails.LocalizedMessage{
+// createPushNoCommand implement logic without command
+func (s *PushService) createPushNoCommand(
+	ctx context.Context,
+	req *pushproto.CreatePushRequest,
+	localizer locale.Localizer,
+	editor *eventproto.Editor,
+) (*pushproto.CreatePushResponse, error) {
+	if err := s.validateCreatePushRequestV2(req, localizer); err != nil {
+		return nil, err
+	}
+	push, err := domain.NewPush(
+		req.Name,
+		string(req.FcmServiceAccount),
+		req.Tags,
+	)
+	if err != nil {
+		s.logger.Error(
+			"Failed to create a new push",
+			log.FieldsFromImcomingContext(ctx).AddFields(
+				zap.Error(err),
+				zap.String("environmentNamespace", req.EnvironmentNamespace),
+				zap.Strings("tags", req.Tags),
```

Comment:
Let's add the `name` to the logs for better debugging.
Same for the other logs.

Reply:
I have added `name` to the logs